.pre-commit-config.yaml

```diff
@@ -87,5 +87,5 @@ repos:
         entry: script/run-in-env.sh python3 -m script.hassfest
         pass_filenames: false
         language: script
-        types: [json]
-        files: ^homeassistant/.+/(manifest|strings)\.json$
+        types: [text]
```

Comment:
@frenck I had to change the pre-commit config here from `json` to `text` or else it would filter out `.coveragerc`. Is there a better way?

Reply:
I think this is fine. With this change, it is not just `.coveragerc` it is now also changes to tests, or the adding of Python files to an integration that can change the outcome.

I guess we could define a `file` filter, but I doubt it would be more efficient at this point.